samples/subsys/mgmt/mcumgr/smp_svr/sysbuild/mcuboot_ram_load.conf

```diff
@@ -0,0 +1,9 @@
+CONFIG_SRAM_SIZE=30
```

Comment:
I think I'm missing something here- why are we manually setting the SRAM size? Shouldn't it still be possible to get this from devicetree if we change the `zephyr,sram` chosen node? Or is the issue that we have set `CONFIG_XIP=n`?

Reply:
It's because for this sample, part of the RAM is used for retention (at the end of RAM - see the overlay) so the amount of RAM needs to be limited for the linker but still available for use with retention

Reply:
Ah ok, so just a specific thing for this sample- not a requirement of MCUBoot RAMload support